index.html

```diff
@@ -90,119 +106,135 @@ <h2>Software Developer</h2>
           </a>
         </li>
       </ul>
-      <div class="work">
+      <div class="work-mobile work">
         <img src="./images/down-arrow.png" alt="" /> <span>work</span>
       </div>
     </div>
+    <div class="work work-desk">
+      <img src="./images/down-arrow.png" alt="" /> <span>work</span>
+    </div>
     <div class="hidden">
       <p>work</p>
     </div>
   </div>
-  <section class="recent-projects">
-    <div class="recent-prj-container">
-      <div class="my-prj">
+
+  <section class="my-recent-works">

```

Comment:
- [ ] Please increase the size of the images and make them look familiar to those of the Figma template.
### Your Implementation
![image](https://user-images.githubusercontent.com/45460192/178035745-7d67c072-d351-4b6c-8ed1-7c4269227d8a.png)
### Figma 
![image](https://user-images.githubusercontent.com/45460192/178035819-24495dbb-135f-4797-8dbc-5958558f5f3b.png)